src/components/Header.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from "react";
+
+export default class Header extends Component {
+
+    state = {
+        block: null
+    }
+
+    getBlockData(){
+        const {blocks,blockShortcut} = this.props;
+        console.log(blocks);
+        let elem = blocks.filter( bl => bl.name.toLowerCase() === blockShortcut.toLowerCase());
```

Comment:
plese use `find` method of array ;)
docs 👉 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find